src/govuk/components/panel/_index.scss

```diff
@@ -12,7 +12,15 @@
     text-align: center;
 
     @include govuk-media-query($until: tablet) {
-      padding: govuk-spacing(6) - $govuk-border-width;
+      padding: govuk-spacing(3) - $govuk-border-width;
+
+      // This is an if-all-else-fails attempt to stop long words from clipping on very narrow
+      // viewports by forcing them to break and wrap instead. The clipping is more likely to happen
+      // when user increases text size on a mobile eg. using iOS Safari text resize controls, and is
+      // more applicable to the panel than other components since the panel padding limits the width
+      // of text more than our other components do.
```

Comment:
I think the main reason this is an issue specifically for the panel component is the fact that it uses white text on a coloured background, so when text overflows outside of the container you end up with white text on the white (page) background which is then invisible.

When most (all?) other components overflow, it can look a bit ugly (and create horizontal scrolling) but the text remains legible.

Is there anything else this component does that means it limits the width of text more than other components? I'm not aware of anything, but might be missing something.

Reply:
Thank you, these are good points 👍  I've updated the code comment to reflect, have a look.